AtReconstruction/AtCopyAuxTreeTask.cxx

```diff
@@ -0,0 +1,89 @@
+#include "AtCopyAuxTreeTask.h"
+
+#include "AtRawEvent.h"
+
+#include <FairLogger.h>
```

Comment:
/user/anthonya/attpcroot/AtReconstruction/AtCopyAuxTreeTask.cxx should add these lines:
#include <TFile.h>            // for TFile
#include <TTree.h>            // for TTree